api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -350,7 +348,35 @@ public void getDateFormat_shouldReturnAPatternWithFourYCharactersInIt() {
 		Assert.assertEquals("tt.MM.uuuu", OpenmrsUtil.getDateFormat(Locale.GERMAN).toLocalizedPattern());
 		Assert.assertEquals("dd-MM-yyyy", OpenmrsUtil.getDateFormat(new Locale("pt", "pt")).toLocalizedPattern());
 	}
+	/**
+	 * @see OpenmrsUtil#getDateTimeFormat(Locale)
+	 */
+	@Test
+	public void getTimeFormat_shouldReturnPatternWithTwoCharactersInIt() {
+		OpenmrsUtil.getDateTimeFormat(Locale.US);
+		OpenmrsUtil.getDateTimeFormat(Locale.UK);
+		OpenmrsUtil.getDateTimeFormat(Locale.GERMANY);
+		OpenmrsUtil.getDateTimeFormat(Locale.forLanguageTag("en"));
+		Assert.assertEquals("hh:mm:ss", "hh:mm:ss");
```

Comment:
@sherrif10  ,that is not the way we test . we test an expected out put and an actual out put from the method being tested